### PR TITLE
New choose session view layout (iPhone SE fix)

### PR DESCRIPTION
**Pull request by pawelgil**
image

### CODE REVIEW

File: AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift [hunk 1]
```diff
@@ -34,238 +34,247 @@ struct ChooseSessionTypeView: View {
...
+                    }
+                )
+                .onAppear { defineNextMove() }
+                .onChange(of: tabSelection.selection, perform: { _ in defineNextMove() })
+                .environmentObject(viewModel.passSessionContext)
         }
     }
     
-    
+    @State private var labelHeight = CGFloat.zero
```
Comment (jankrzempek) [inline AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift:223]: Maybe we could put this state next to the others variables, what do you think?

File: AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift [hunk 2]
```diff
@@ -34,238 +34,247 @@ struct ChooseSessionTypeView: View {
...
     private var mainContent: some View {
-        VStack(spacing: 50) {
-            VStack(alignment: .leading, spacing: 10) {
-                titleLabel
-                messageLabel
-            }
-            .padding(.horizontal)
+        GeometryReader { geometry in
+            let minimalRequiredHeight = 499.0
```
Comment (jankrzempek) [inline AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift:226]: Maybe we could add : `Based on the investigation carried on iPhone SE 1st` ?

**Review by jankrzempek — APPROVED** [review]
🥳

**Review by anna1901 — APPROVED** [review]

File: AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift (hunk 1 above)
Comment (pawelgil) [reply AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift:223]: I also changed the name to `buttonHeight` to be more descriptive

**Pull request merged**